src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiConverter.shared.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiConverter : IMultiValueConverter
```

Comment:
Should it implement Converter Extension somehow (perhaps we will have to create a new extension class for multy converter)